setup.py

```diff
@@ -8,6 +8,12 @@
 
 import bravado
 
+fido_client_dependencies = [
```

Comment:
no need for unnecsesary indirection, just inline these.  It's already pretty obvious these are for "fido" since they are under the fido block.

Why do you need an explicit dependency on crochet / yelp-bytes, shouldn't fido be handling that?